contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+
+For example, imagine you are filling out a complex form on a website with many input fields, but the form is slow to respond when you try to interact with it. Even after the page has finished loading, the validation scripts that check if you've entered the correct information might still be running in the background, causing the form to be slow or unresponsive. TTI measures how long it takes for the page to become fully interactive and responsive to user inputs.
+
+While there is no benchmark for what is a good time for Page Loaded, a [good time for TTI](https://developer.chrome.com/docs/lighthouse/performance/interactive/#how-lighthouse-determines-your-tti-score) is under 3.8 seconds while over 7.3 seconds is considered slow.
+
+
+## So what's more important? FCP, Dom Interactive, or Page Loaded?
+All of them are important for a good user experience! However, you may decide to focus on a specific metric depending on your goal. For example:
+
+- For a page with a large banner image above the fold, a high FCP time means that the user has to wait too long before any visual content appears on the screen.
+- If your goal is to improve your [PageSpeed Insights](https://pagespeed.web.dev/) performance score, then Page Loaded will be the most important since this contributes the most to [Total Blocking Time](https://web.dev/tbt/), which has the [highest weighting](https://developer.chrome.com/docs/lighthouse/performance/performance-scoring/?utm_source=lighthouse&utm_medium=lr#weightings) when calculating the performance score.
+- If you're building an e-commerce site with many input fields and complex validation, then a high DOM Interactive time means that the form is slow to respond to user input. 
+
+
+## Optimization cheat sheet
+If you're looking to improve any of the above metrics, here is a handy cheat sheet for where to look for opportunities. Each value in the table indicates how likely the metrics are to be affeted.
+
+| What?   | First Contetful Paint  | DOM Interactive | Page Loaded |
+|--------|--------|--------|--------|
+| Page Size   | 🟢 High  | 🟢 High   | 🟢 High |
+| Server Response Time   | 🟢 High  | 🟢 High | 🟢 High |
+| Network Latency   | 🟢 High  | 🟢 High | 🟢 High |
+| Downloading Javacript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+| Executing Javacript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+| CSS Files: Downloading and Parsing  | 🟢 High | 🟡 Medium | 🟡 Medium |
+| Images: Downlading and Rendering| 🟡 Medium (only if they are part of the initial content) | 🔴 Low | 🟢 High |
+| Fonts: Downloading and Rendering  | 🟡 Medium (only if they are part of the initial content)  | 🔴 Low |  🔴 Low  |
+
+## Identifying Opportunities on Your Own Site
```

Comment:
I felt like I got to this section and then the article ended a bit abruptly - I wonder if it would be worth expanding on this a lot more in terms of the kinds of places people should check? (I imagine there are a typical 5-10 things that are always the biggest contributors, followed by a long tail of small stuff?)

Reply:
+1